app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
+
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
```

Comment:
let's follow the first option from checklist where we move all parameters on the next lines

Reply:
you mean in this case everything is good? like good example?

Reply:
Ihor suggested you to use first example form the checklist in Linter formatter task, since this type of formatting is more common
https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md